src/Compilers/Core/Private/ObsoleteAttribute.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum |
+        AttributeTargets.Interface | AttributeTargets.Constructor | AttributeTargets.Method |
+        AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Delegate,
+        Inherited = false)]
+    internal sealed class ObsoleteAttribute : Attribute
```

Comment:
Manually adding new definition of `ObsoleteAttribute` which has `DiagnosticId` so the warning can be disabled with one line in `.globalconfig`. That avoids lots of warning suppressions across the Roslyn repo (plus might be useful for customers).